src/include/cephfs/metrics/Types.h

```diff
@@ -60,16 +60,32 @@ inline std::ostream &operator<<(std::ostream &os, const ClientMetricType &type)
   return os;
 }
 
-struct CapInfoPayload {
-  static const ClientMetricType METRIC_TYPE = ClientMetricType::CLIENT_METRIC_TYPE_CAP_INFO;
+struct Payload {
+  Payload() = delete;
```

Comment:
no need to delete the default ctor explicitly as you already have another ctor which accepts parameters.

Reply:
Sure, will fix it.